docs/api.md

```diff
@@ -0,0 +1,1425 @@
+
+# API
+
+- all paths should be specified treating the user's home directory as root
+
+- The url base, in an actualy deployment, will be:
```

Comment:
typo

Reply:
Fixed this and about a half dozen other typos.